Aims.Sdk/Properties/AssemblyInfo.cs

```diff
@@ -32,5 +31,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+//[assembly: AssemblyFileVersion("1.0.0.0")]
```

Comment:
Do we need this line? Usually we don't keep code commented out unless there's a good reason.